IntelliTect.TestTools.WindowsTestWrapper/IntelliTect.TestTools.WindowsTestWrapper/Utilities.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+
+namespace IntelliTect.TestTools.WindowsTestWrapper
+{
+    public class Utilities
+    {
+        /// <summary>
+        /// Parse a text file and return a string of its contents
+        /// </summary>
+        /// <param name="path">The full path of the location and file name</param>
+        /// <returns>Returns the text of the chosen file</returns>
+        public string ParseTextFile(string path)
+        {
+            string text = null;
+            int seconds = 0;
```

Comment:
You probably want to name this something like `retryAttempts` since it is not actually a measurement of seconds.